misc/docs-linter/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/gnolang/gno/tm2/pkg/commands"
+	"golang.org/x/sync/errgroup"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+)
+
+var (
+	ErrEmptyPath = errors.New("you need to pass in a path to scan")
+)
+
+type cfg struct {
+	docsPath string
+}
+
+func main() {
+	cfg := &cfg{}
+
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			Name:       "docs-linter",
+			ShortUsage: "docs-linter [flags]",
+			ShortHelp:  "Finds broken 404 links in the .md files in the given folder & subfolders",
+		},
+		cfg,
+		func(ctx context.Context, args []string) error {
```

Comment:
It's funny that the go linter was not run against this code

Reply:
I ran the linter and fixed the unused variables: https://github.com/gnolang/gno/pull/2125/commits/fe69c6a67fee0d915b2df3334733e55208a34ebf

Reply:
@zivkovicmilos, i think that the CI isn't running gofumpt against "misc/".